tests/integration/test_design_import.py

```diff
@@ -170,6 +171,34 @@ def test_open_file(modeler: Modeler, tmp_path_factory: pytest.TempPathFactory):
     sketch = Sketch(Plane(Point3D([0, 5, 5]))).box(Point2D([5, 2.5]), 10, 5)
     comp1.extrude_sketch("Top", sketch, 5)
 
+    modeler.unsupported.set_export_id(base_body.id, PersistentIdType.PRIME_ID, "1")
+    modeler.unsupported.set_export_id(wheel_body.id, PersistentIdType.PRIME_ID, "2")
+
+    modeler.unsupported.set_export_id(base_body.faces[0].id, PersistentIdType.PRIME_ID, "3")
+    modeler.unsupported.set_export_id(base_body.edges[0].id, PersistentIdType.PRIME_ID, "4")
+
+    bodies1 = modeler.unsupported.get_body_occurrences_from_import_id(
+        "1", PersistentIdType.PRIME_ID
+    )
+    bodies2 = modeler.unsupported.get_body_occurrences_from_import_id(
+        "2", PersistentIdType.PRIME_ID
+    )
+
+    # requires a change to core service, uncomment on next core service update
+    # assert base_body.id in [b.id for b in bodies1]
+    # assert wheel_body.id in [b.id for b in bodies2]
+    assert base_body.id not in [b.id for b in bodies1]
+    assert wheel_body.id not in [b.id for b in bodies2]
+
```

Comment:
This is going to prevent the update of the Docker images unless we do it right away.... we are checking that the new images are not breaking the tests before promoting them. This is done on ``main`` rather than on ``blitz``. And we are safe because of that but nonetheless, it is going to be problematic on the workflows